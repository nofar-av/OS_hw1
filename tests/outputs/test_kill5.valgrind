==28577== Memcheck, a memory error detector
==28577== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==28577== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==28577== Command: /home/omer/OS_wet1/smash
==28577== Parent PID: 28573
==28577== 
==28577== 
==28577== FILE DESCRIPTORS: 4 open at exit.
==28577== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_kill5.valgrind
==28577==    <inherited from parent>
==28577== 
==28577== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_kill5.err
==28577==    <inherited from parent>
==28577== 
==28577== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_kill5.out
==28577==    <inherited from parent>
==28577== 
==28577== Open file descriptor 0:
==28577==    <inherited from parent>
==28577== 
==28577== 
==28577== HEAP SUMMARY:
==28577==     in use at exit: 0 bytes in 0 blocks
==28577==   total heap usage: 49 allocs, 49 frees, 82,620 bytes allocated
==28577== 
==28577== All heap blocks were freed -- no leaks are possible
==28577== 
==28577== For counts of detected and suppressed errors, rerun with: -v
==28577== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
