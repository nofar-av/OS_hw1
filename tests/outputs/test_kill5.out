smash> smash> smash> signal number 9 was sent to pid 28736
smash> smash: got ctrl-Z
